docs/posts/20210427_ICU4X_02_Release.md

```diff
@@ -0,0 +1,28 @@
+# ICU4X 0.2 Released
+
+Unicode® ICU4X 0.2 has just been released.
+
+The revision improves completness of the components introduced in ICU4X 0.1 and introduces a number of lower level utilities.
+
+## Components
+
+In 0.2, ICU4X improves `DateTimeFormat` adding initial support for time zones, component bags and day periods. A new `LocaleCanonicalizer` component has been added to support `Locale` data driven operations. Third improved component is `PluralRules` which gained support for the `E` operand.
+
+## Utilities
+
+With a small set of core components maturing, the project starts to focus more on the features we want to develop for all components such as strong FFI support and flexible data management.
+
+For that purpose, we started building a number of lower level utilities focused on bringing high performance data structures and algorithms.
+
+To minimize the size of our WASM binaries we added `Writeable`, which introduces a lightweight alternative to `Display` and `LiteMap`, which adds a `HashMap` replacement that uses `binary_search` instead of carrying with it a hashing logic.
```

Comment:
```suggestion
To minimize the size of our WASM binaries we added `Writeable`, which introduces a lightweight alternative to `Display` and `LiteMap`, which adds a `HashMap` replacement that uses `binary_search` instead of a hash function.
```